src/main/java/org/synyx/urlaubsverwaltung/account/AccountSettings.java

```diff
@@ -18,6 +25,13 @@ public class AccountSettings {
      */
     private Integer maximumAnnualVacationDays = 40;
 
+    /**
+     * Specifies the date, when the vacation will expire globally
+     */
+    @Convert(converter = MonthDayDateAttributeConverter.class)
+    @Column(name = "account_expiry_date")
+    private MonthDay expiryDate = MonthDay.of(APRIL, 1);
```

Comment:
Weshalb in der Entity ein `MonthDay` und den Converter dafür, der das dann in ein `LocalDate` mit Jahr `1` konvertiert?

Statt zwei Felder, eins für den Tag, eins für den Monat?

Für Services kann die Entity einen getter spendiert bekommen für das `MonthDay`. Dann ist das dort angenehmer mit zu arbeiten.

Im User Interface hätte ich auch zwei Eingabefelder angeboten. Ein Input für den Tag, eine Select für den Monat. Da muss nichts fancy gebaut werden mit JavaScript und die Eingabe ist imho auch die einfachste und deutlichste.

Reply:
Ist schon bisschen her, aber ich meine ich habe das "einfach mal" runter programmiert um zu schauen an welchen Stellen wir was anfassen müssen. Wir können die Informationen/Datentypen gerne anpassen, wenn andere besser passen.